src/main/java/io/kabanero/api/InstanceEndpoints.java

```diff
@@ -124,4 +127,33 @@ public Response isAdmin(@PathParam("instanceName") String instanceName) throws I
         return Response.ok(body).build();
     }
 
-}
\ No newline at end of file
+    @GET
+    @Path("{instanceName}/team/{wantedTeamName}")
+    @Produces(MediaType.APPLICATION_JSON)
+    public Response isAdmin(@PathParam("instanceName") String instanceName, @PathParam("wantedTeamName") String wantedTeamName) throws IOException, ApiException, GeneralSecurityException {
+        UserProfile userProfile = UserProfileManager.getUserProfile();
+        String token = userProfile.getAccessToken();
+        GitHubClient client = new GitHubClient();
+        client.setOAuth2Token(token);
+
+        Kabanero instance = KabaneroClient.getAnInstance(instanceName);
+        if (instance == null) {
+            return Response.status(404).entity(new ResponseMessage(instanceName + " not found")).build();
+        }
+
+        String instanceGithubOrg = instance.getSpec().getGithub().getOrganization();
+        TeamService teamService = new TeamService(client);
+
+        for (Team orgTeam : teamService.getTeams(instanceGithubOrg)) {
+            if (wantedTeamName.equals(orgTeam.getName())) {
+                List<User> kabaneroTeamMembers = teamService.getMembers(orgTeam.getId());
+                if (kabaneroTeamMembers.size() == 0) {
+                    return Response.status(404).entity(new ResponseMessage(wantedTeamName + " currently does not have any members")).build();
+                }
```

Comment:
not sure 404 would make sense (if the team didn't exist it would make sense) but in this case we did find the team.

No need to send a specific code back if its empty, just send it back even if its empty, doesn't matter. So I would delete this `if`